packages/wavs/src/main.rs

```diff
@@ -37,15 +41,40 @@ fn main() {
         None
     };
 
-    ctx.rt.block_on(async {
-        // warn bad health for chains (services may or may not submit to these)
-        let chains = config.chains.all_chain_keys().unwrap();
-        if !chains.is_empty() {
-            if let Err(err) = health_check_chains_query(&config.chains, &chains).await {
-                tracing::warn!("Non-trigger-chain health-check failed: {}", err);
+    let health_status = SharedHealthStatus::new();
+
+    let chains = config.chains.all_chain_keys().unwrap();
+    if !chains.is_empty() {
+        match config.health_check_mode {
+            HealthCheckMode::Bypass => {
+                let health_status_clone = health_status.clone();
+                let chain_configs = config.chains.clone();
+                ctx.rt.spawn(async move {
+                    tracing::info!("Running health checks in background (bypass mode)");
+                    health_status_clone.update(&chain_configs).await;
```

Comment:
this should also log, like `Wait`, only difference is it logs in the background whenever it happens to run, we don't wait on it